src/main/resources/post_processing/post_process_provenance_appointment.sql

```diff
@@ -0,0 +1,26 @@
+DO
+$$
+DECLARE v_rowCount int;
+BEGIN
+update cds_cdm.appointment as app
+set visit_occurrence_id = app_prov_details.visit_occurrence_id,
+person_id = app_prov_details.person_id,
+care_site_id = app_prov_details.care_site_id
+	from (
+		with qr_details as (
+			select * from cds_etl_helper.post_process_map ppm where "type" = 'QUESTIONNAIRERESPONSE' and data_one <> 'ENCOUNTER'
+		),
+		provenance_details as (
+			select * ,
+			split_part(data_two, ';', 2) as questionnaire_response_logical_id,
+			split_part(data_two, ';', 1) as appointment_logical_id
+			from cds_etl_helper.post_process_map ppm where "type" = 'PROVENANCE' and data_one = 'Appointment'
```

Comment:
Please check if Appointment should be APPOINTMENT to maintain uniformity in variable naming.

Reply:
date_one is supposed to be in lowercase in the post_process_map table because it can also contain not only texts but other type of data like date and date time.